src/app/Authenticated.tsx

```diff
@@ -192,6 +194,12 @@ const Authenticated = () => {
                             path={routeDetails.admin.path}
                             element={<Admin />}
                         />
+                        {!isProduction && (
+                            <Route
+                                path="test/jsonforms"
+                                element={<TestJsonForms />}
+                            />
+                        )}
```

Comment:
Making sure this does not show in Production... even though there isn't much risk if it did.